xcode/Subconscious/Shared/Components/Detail/Detail.swift

```diff
@@ -1455,38 +1455,39 @@ struct DetailModel: ModelProtocol {
         environment: AppEnvironment,
         audience: Audience
     ) -> Update<DetailModel> {
-        guard let address = state.address else {
+        guard let from = state.address else {
             logger.log(
                 "Update audience requested, but no memo is being edited."
             )
             return Update(state: state)
         }
-        let fx: Fx<DetailAction> = environment.data.updateAudienceAsync(
-            address: address,
-            audience: audience
+        let to = from.withAudience(audience)
+        let fx: Fx<DetailAction> = environment.data.moveEntryAsync(
```

Comment:
With these new MemoAddress semantics, changing audience becomes a kind of move operation (it was before, too, but this was obscured by the types).